app/index.js

```diff
@@ -18,6 +18,9 @@ render(
   document.getElementById('root')
 );
 
+document.addEventListener('dragover', event => event.preventDefault());
+document.addEventListener('drop', event => event.preventDefault());
```

Comment:
@anonymoussprocket what are these for?

Reply:
prevents drag/drop action into the electron container. presently if you drop a document into the host window, it will get loaded and there's no way to get back to the app.

Reply:
![drag-drop](https://user-images.githubusercontent.com/37525692/48863796-a0008280-ed98-11e8-81fc-74ce73a1638f.gif)